<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loan</class>
 <widget class="QDialog" name="Loan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 233, 235);</string>
  </property>
  <widget class="QTreeWidget" name="treeWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>211</width>
     <height>171</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save and Proceed to give Loan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>89</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>148</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Date Due</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>LOAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>clear list</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>159</y>
     <width>271</width>
     <height>250</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Savings</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>179</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Double click to choose surtee (max 3)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Loan</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
